index.html

```diff
@@ -107,11 +112,69 @@ <h3><a href="https://github.com/FIRSTMap/firstmap.github.io">View or edit FIRSTM
             </div>
         </div>
 
+        <div id="search">
+            <h2 class="stripe">Search</h2>
+
+            <div id="searchSection" hidden>
+                <h3>Search</h3>
+                <div id="teamSearch">
+                    <input type="search" id="teamSearchInput" onkeyup="inputKeyUp(this)" />
+                    <button onclick="search()">Search</button>
+                </div>
+
+                <div id="eventSearch" hidden>
+                    <input type="search" id="eventSearchInput" />
+                    
+                    <ul id="eventSearchList">
+                    </ul>
+                </div>
+
+                <select id="searchType" onchange="(() => { updateSearchType(); })()">
```

Comment:
I know we do it elsewhere but it's not great practice to put JS code in `on`- HTML properties.